README.md

```diff
@@ -130,10 +130,15 @@ Use the "Table on Contents" menu on the top-left corner to explore the list.
 - [vite-plugin-handlebars](https://github.com/alexlafroscia/vite-plugin-handlebars) - Process HTML files with Handlebars.
 - [vite-plugin-html-config](https://github.com/ahwgs/vite-plugin-html-config) - HTML tag configuration, such as script link style meta favicon and more.
 - [vite-plugin-html-template](https://github.com/IndexXuan/vite-plugin-html-template) - HTML template for vite app, like html-webpack-plugin for webpack.
+- [vite-plugin-transform](https://github.com/Silksofthesoul/vite-plugin-transform) - Plugin resolve paths in cases where standard ways dont help, and replacement functionality.
```

Comment:
## Checklist

- [ ] git: pull request template structure not broken
- [ ] git: pull request template not selected `type`
- [ ] git: pull request template not checked `Checklist`
- [ ] git: empty commit
- [ ] git: link not to repository
- [ ] git: link description contains a link to the author / third-party resource
- [ ] git: broken PR
- [ ] git: no conflicts resolved
- [ ] git: doublicate
- [ ] time: not updated for more than 6 months
- [ ] category: not in its category
- [x] lists: not the last on the list
- [ ] docs: no documentation
- [ ] docs: non-english documentation
- [ ] project: commercial product
- [ ] project: dead project
- [ ] project: does not accept donat
- [ ] site: no link to the site
- [ ] site: site with errors
- [ ] site: not actively using vuejs
- [ ] site: not original
> Follow the rules [CONTRIBUTING](https://github.com/vitejs/awesome-vite/blob/master/.github/contributing.md)

Reply:
@Scrum fixed